Source/Core/Core/HW/SI_DeviceGCController.cpp

```diff
@@ -26,6 +26,26 @@ CSIDevice_GCController::CSIDevice_GCController(SIDevices device, int _iDeviceNum
 {
 	// Dunno if we need to do this, game/lib should set it?
 	m_Mode                   = 0x03;
+
+	m_Calibrated = false;
+}
+
+void CSIDevice_GCController::Calibrate()
+{
+	GCPadStatus pad_origin;
+	memset(&m_Origin, 0, sizeof(SOrigin));
+
+	pad_origin = GetPadStatus();
```

Comment:
the variable can be joined with the declaration here, other than that, it looks fine to me.